scripts/README.md

```diff
@@ -15,19 +15,20 @@ For example, to point all links to innersourcecommons.org:
 node substitute_article_urls.js isc
 ```
 
-## generate_learning_path_markdown.js
+## generate_old_site_learning_path_markdown.js, generate_new_site_learning_path_markdown.js
```

Comment:
For future compatibility probably good to label with something besides `old` and `new` (`jekyll` and `hugo`?).  I've painted myself into a corner before with terminology like this one once the `new` website becomes the old one later on.

Reply:
Good call, jekyll/hugo make more sense.